core/src/main/java/org/apache/iceberg/CatalogProperties.java

```diff
@@ -30,6 +30,21 @@ private CatalogProperties() {
   public static final String FILE_IO_IMPL = "io-impl";
   public static final String WAREHOUSE_LOCATION = "warehouse";
 
+  public static final String CACHE_ENABLED = "cache-enabled";
+  public static final boolean CACHE_ENABLED_DEFAULT = true;
+
+  /**
+   * Behavior of specific values of cache.expiration-interval-ms
+   * <ul>
```

Comment:
nit: need a newline using `<p>`

Reply:
You mean a newline before the unordered list tag?

I don't think this needs <p>. Here's the docs generated without it.

![image](https://user-images.githubusercontent.com/9833362/145908476-771b18a2-cd50-441c-9412-42d82f0c24c0.png)